### PR TITLE
Use `error_msg` instead of `msg_format`

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we migrate from using the old `msg_format` to the new `error_msg`, which has the updated doc, describing the formatting tokens supported by the Validation library.

### CODE REVIEW

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/Update.kt [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        if (!sshConfigFile.exists()) {
+            val parentDir = sshConfigFile.canonicalFile.parentFile
+            parentDir.mkdirs()
+            sshConfigFile.createNewFile()
+        }
+        return sshConfigFile
+    }
+
+    /** Executes a command in the project [rootFolder]. */
+    private fun execute(vararg command: String): String = Cli(rootFolder).execute(*command)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/Update.kt:196]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        updatePagesTask.configure {
+            dependsOn(copyJavadoc)
+        }
+    }
+
+    private fun TaskContainer.registerCopyJavadoc(allowInternalJavadoc: Boolean) {
+        val inputs = composeInputs(allowInternalJavadoc)
+        register(copyJavadoc, Copy::class.java) {
+            doLast {
+                from(*inputs.toTypedArray())
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt:151]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/fs/LazyTempPath.kt [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    override fun resolveSibling(other: String): Path = delegate.resolveSibling(other)
+
+    override fun relativize(other: Path): Path = delegate.relativize(other)
+
+    override fun toUri(): URI = delegate.toUri()
+
+    override fun toAbsolutePath(): Path = delegate.toAbsolutePath()
+
+    override fun toRealPath(vararg options: LinkOption?): Path = delegate.toRealPath(*options)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/fs/LazyTempPath.kt:111]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt [hunk 4]
```diff
@@ -26,9 +26,71 @@
...
+     *
+     * @param command the command to execute
+     * @return the command line output
+     * @throws IllegalStateException upon an execution error
+     */
+    fun execute(vararg command: String): String {
+        val outWriter = StringWriter()
+        val errWriter = StringWriter()
+
+        val process = ProcessBuilder(*command)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt:63]: ignore

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Review by armiol — APPROVED** [review]

**Pull request merged**